src/drivers/src/i2c_drv.c

```diff
@@ -498,6 +498,7 @@ static void i2cdrvEventIsrHandler(I2cDrv* i2c)
       // Enable the Transfer Complete interrupt
       DMA_ITConfig(i2c->def->dmaRxStream, DMA_IT_TC | DMA_IT_TE, ENABLE);
       I2C_DMACmd(i2c->def->i2cPort, ENABLE); // Enable before ADDR clear
+      DMA_Cmd(i2c->def->dmaRxStream, ENABLE); // Workaround for emulation...
```

Comment:
All information about this workaround will be lost to time, only myths and legends will remain. Could you possibly add a, somewhat, more detailed comment about why this was needed or what problems you saw without it?

In other words, something like
```
//
//
//
//
DMA_Cmd(i2c->def->dmaRxStream, ENABLE);
```
Rather than:
```
DMA_Cmd(i2c->def->dmaRxStream, ENABLE); //
```